### ISSUE TITLE
TrainTrack: Help transitioning from Python 2 to 3

### ISSUE THREAD

**Issue by raamana** [ohbm/hackathon2019#25]
# Name of your awesome project
2to3: Porting your package from python 2 to 3

## Project Description
As the Python 2 is reaching end of life, the need to transition to Python 3 is imminent and important. Hence, this tutorial would help orient those who need to, with the necessity, sufficient guidance and discussion on issues related to it.

## Skills required to participate
- Basic python skills
- Patience to debug and test

## Integration
I am yet to define the full scope of this (whether I have time to do this myself). Will update this soon with all the details.

TBA

## Preparation material

Tagged under migration [here](https://github.com/ohbm/hackathon2019/blob/master/Tutorial_Resources.md#specific-topics)

## Link to your GitHub repo

TBA

## Communication

TBA

**Comment by effigies** [comment]
Depending on how long each TrainTrack gets, it might make sense to try to make the presentation part as short as possible and spend most of it doing hands-on assistance.

If that wouldn't be enough time to make a reasonable start, perhaps we could try to do this as early as possible, and then have a portion of the hack space marked for people who want to work on migration.

**Comment by raamana** [comment]
Yes - this presentation would be ideal to delineate the differences, present HowTos, and some pointers to tricky issues, and help them design tests to ensure the transition was accurate etc. Actual hacking for each participant would not be feasible in the TrainTrack setting I think.
